### PR TITLE
change image name to debian8 for jessie

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -14,6 +14,6 @@ steps:
         - name: gcr.io/cloud-builders/docker
           args:
                 - 'build'
-                - '--tag=%REPO%/debian:%TAG%'
+                - '--tag=%REPO%/debian%NUM%:%TAG%'
```
Comment (dlorenc) [inline cloudbuild.yaml]: We're getting a few of these now, would you mind adding some comments to the top of this yaml file explaining what each of the variables are?

File: build.sh [hunk 2]
```diff
@@ -10,9 +10,15 @@ then
   GCLOUD_CMD=$3
 fi
 
+if [ "$VERSION" == "jessie" ]
```
Comment (dlorenc) [inline build.sh]: Should we error if we don't have an if statement for the VERSION variable?

Comment (sharifelgamal) [inline build.sh]: Agreed, done.

File: cloudbuild.yaml (hunk 1 above)
Comment (sharifelgamal) [inline cloudbuild.yaml]: Done

**Pull request closed without merging**